server/controllers/base/formInitialStep.ts

```diff
@@ -74,4 +129,70 @@ export default class FormInitialStep extends FormWizard.Controller {
   formError(fieldName: string, type: string): FormWizard.Controller.Error {
     return new FormWizard.Controller.Error(fieldName, { args: {}, type, url: '/' })
   }
+
+  setupDateInputFields(fields: FormWizard.Fields, errorlist: FormWizard.Controller.Error[]): FormWizard.Fields {
```

Comment:
we already had some date inputs previously, what's this new function intended to do on top of what already happened before?

Reply:
So I did mean to leave this out actually, so I can go back and remove it. However, they used it as it related to the gov date input fields, where it splits day, month and year into separate input boxes. They are rather fiddly to deal with and validate etc. I tried playing about with it when I first was bringing in the form wizard, they have added a bunch of functions including validation checks etc to deal with them. They were told they had to use the date inputs as part of their designs, if we can use the date picker instead (as we have done so far) it makes our lives easier and I don't think we need these functions. So happy to take it out or can leave in in case we need it in future.

Reply:
i would prefer to leave it out for now because we don't have tests for it and we don't know if we will ever use it should UCD go down the route of date pickers

Reply:
this has been removed now